commands/games.py

```diff
@@ -8,8 +8,14 @@ class C_Games(Command):
     @classmethod
     async def run(self,ctx):
         page = 0
-        if ctx.mentions:
-            u = db.User.from_mem(ctx.mentions[0])
+        a = -1 # Not found
+        if len(ctx.args) > 0:
+            if (len(ctx.mentions) == 0):
+                a = int(ctx.args[0])
+            else:
+                a = int(ctx.mentions[0].id)
+        if db.User.from_id(int(a)).exists:

```

Comment:
No need to have 2 db operations here when you could have one. Also .exists is not needed. If exists is false the entire object is falsey